### ISSUE TITLE
Scss config / rules

### ISSUE THREAD

**Issue by saltcod** [tierratelematics/code-beautifier#1]
Hey there!

This ext seems to work great out of the box. I'm wondering if there's anywhere I can add some rules? I'd just like to add some spaces after nested elements. Example: 

Instead of this: 

```
.selector {
margin-top: 5px;
a {
padding 10px; 
&:after { style }
}
}
```

I'd love to see line breaks after the nested parts:

```
.selector {
margin-top: 5px;

a {
padding 10px; 

&:after { style }
}
}
```

Thanks very much for your time and efforts with this!

**Comment by saltcod** [comment]
That didn't format things too well, so here's a quick screenshot to make sure I'm coming across clearly:

https://dl.dropboxusercontent.com/s/vn80fwk1vc5f4df/2016-10-24%20at%2011.19%20PM.png

**Comment by langdonx** [comment]
@saltcod I'd like some more double spacing myself.  Have you found any alternatives?

**Comment by mike7515** [comment]
Fixed in the last release 1.7.2
